pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -118,9 +118,11 @@ func coreBootstrap(initConfiguration *kubeadmapi.InitConfiguration, bootstrapCon
 		return errors.Wrap(err, "error writing init configuration")
 	}
 
-	var criConfigure string
-	if _, err := os.Stat(skuba.CriDockerDefaultsConfFile()); err == nil {
-		criConfigure = "cri.configure"
+	var criSetup string
+	if _, err := os.Stat(skuba.CriDefaultsConfFile()); err == nil {
```

Comment:
it makes me sad to have this kind of wiring tbh, but I don't see a different approach possible, as we don't have a cluster-init cleaner, and we don't send the files smartly. LGTM.